.claude/skills/release/SKILL.md

```
@@ -106,12 +106,43 @@ work, put the changelog in that pull request and let review happen there.
+commit and no tag. That is fine — make the commit yourself, with cz's own
+message format, because `publish.yml` triggers on a head commit that starts
+with `bump:`:
+
+```bash
+git commit -am "bump: version <previous> → <version>"   # re-run if a hook edits files
+```
+
+Check what you are committing. `cz bump` leaves `uv.lock` alone, so bump the
+`geoparquet-io` package version in it by hand — one line:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)